src/test/java/de/butbetter/keksys/CalculationValidatorTest.java

```diff
@@ -0,0 +1,26 @@
+package de.butbetter.keksys;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest
+class CalculationValidatorTest {
+
+    @Autowired
+    private CalculationValidator calc;
+
+    @Test
+    void calculateNegativeNumber() {
+        assertThrows(IllegalArgumentException.class, () -> calc.calculateVAT(-50, 19));
+        assertThrows(IllegalArgumentException.class, () -> calc.calculateVAT(100, -256));
+    }
+
+    @Test
+    void calculateMax() {
```

Comment:
Error: 2021-11-07 19:07:10.076  INFO 9692 --- [           main] de.butbetter.keksys.VATCalculator        : 3.4028235E38 + 1.0 = Infinity

Reply:
Error was caused by unexpected Exception and is resolved in e42deb3